src/smokeTest/resources/application.properties

```
@@ -0,0 +1,5 @@
+env=${ENVIRONMENT_NAME:local}
+url=${TEST_URL:http://localhost:9000}
```

this should be accessed as a spring property test.url, env override is automatic based on convention
```suggestion
test.url=http://localhost:8080
```